content/articles/tidyverse_1.2.0.Rmd

```diff
@@ -0,0 +1,75 @@
+---
+title: tidyverse 1.2.0
+slug: tidyverse-1.2.0
```

Comment:
Does the underscore in the filename versus the hypen here in the slug explain why the post isn't showing up in the netlify preview (?).

Reply:
Just changed it to see. Thanks, Jenny.